### ISSUE TITLE
[BUG] no work in pdf when admonitions and this plugin are enabled

### ISSUE THREAD

**Issue by mfdycs** [mugiwara85/CodeblockCustomizer#109]
When admonitions and this plugin are enabled, this plugin will not work in PDF.

**Comment by mugiwara85** [comment]
Yes, you are right. It seems, I accidentally broke this feature. I will look into it, and get back to you.

**Comment by zhuolisam** [comment]
For me, even I disabled admonitions the codeblocks line highlights (`hl:` syntax) are not showing after pdf export, but inline code styling works fine. I'm using solarized theme.

**Comment by shaunhenderson-fssi** [comment]
I only have this plugin installed. Line numbers show in the live preview, but not on the PDF

**Comment by mugiwara85** [comment]
@zhuolisam `hl` highlights do not show at all? Or they do not show up in admonitions? What theme are you using? Can you please send a picture about the issue?
@shaunhenderson-fssi What theme are you using? Could you please also send a picture about the issue?

**Comment by shaunhenderson-fssi** [comment]
Here are my settings, I am using "Monokai" for Obsidian and "Obsidian" for the plugin:

![Image](https://github.com/user-attachments/assets/15728d64-9e66-4a7d-beda-e0cb71adffdc)

![Image](https://github.com/user-attachments/assets/e3cdf2a5-96c9-41b7-98b9-f74cb7812615)

Final PDF:

![Image](https://github.com/user-attachments/assets/02bad440-bdf7-41d3-a0a9-8a000ed9b4f7)

**Comment by mugiwara85** [comment]
@shaunhenderson-fssi There is a setting on the `Print to PDF` settings page to `Enable Print to PDF`. Did you enable that option?

**Comment by shaunhenderson-fssi** [comment]
I just went Ctrl+P, then chose the Export to PDF option. The dialog looks like this:

![Image](https://github.com/user-attachments/assets/536b3196-1a8d-4036-9b6f-406375017543)

**Comment by mugiwara85** [comment]
> I just went Ctrl+P, then chose the Export to PDF option. The dialog looks like this:
> 
> ![Image](https://github.com/user-attachments/assets/536b3196-1a8d-4036-9b6f-406375017543)

Not there. In the plugins settings.

![Image](https://github.com/user-attachments/assets/9a6ed6cc-7a0f-4e82-a9fb-21d5fc747510)

**Comment by shaunhenderson-fssi** [comment]
Ohhhhh. Yes I can see that option and enabling it DOES print line numbers on the PDF. Maybe this should be enabled by default? The main reason I am using this plugin is for the line number functionality

**Comment by mugiwara85** [comment]
You are right. It should be enabled by default. Apperantly, when I added this option I forgot to set it to true. But I changed it, so for new installations it will be enabled by default.
The plugin can do much more, not just line numbers ;)

**Comment by mugiwara85** [comment]
Fixed in version 1.3.0
